jest.config.js

```diff
@@ -1,7 +1,8 @@
 'use strict';
 
 /*! m0-start */
-module.exports = {
-    setupFilesAfterEnv: ['./jest.setup.js']
+const config = {
```

Comment:
The stuff inside the comments has been modified manually as some point and will likely get overwritten by the base config. The contents of this file should be:

https://github.com/CriminalInjuriesCompensationAuthority/q-base-config/blob/master/blocks/jest.config.js 

This can the be extended outside of the comments.

Is "setupFilesAfterEnv" no longer used? If so, can the "jest.setup.js" be deleted?